grow/pods/documents.py

```diff
@@ -264,6 +264,13 @@ def locales(self):
             if codes is None:
                 return []
             return locales.Locale.parse_codes(codes)
+        # If `$localization` is missing, use locales inferred from document
```

Comment:
@stucox pinging on the above. (Noticed you weren't cced explicitly in my message, in case you missed it here it is.)